### PR TITLE
sql file complete?

### PR DESCRIPTION
**Pull request by aprilrochelle**
Please structure your pull requests this way:

- **Purpose of PR**
To have team review for Bangazon database setup

- **Issue number this request impacts** ------
#17 

- **Description of what was changed:**

Syntax converted from SQL Lite.

- **Thorough list of steps to test**

Copy SQL file.
Create new query in Bangazon database in SSMS.
Paste SQL code.
Click Execute.

if you don't do this you will be
**FIRED**

### CODE REVIEW

**Comment by Crashofthemoons** [comment]
Tested SQL code in SSMS successfully. I approve this merge

File: BangazonDB.sql [hunk 1]
```diff
@@ -0,0 +1,552 @@
...
+select 'Food';
+
+CREATE TABLE Product (
+Id	integer NOT NULL PRIMARY KEY IDENTITY,
+ProductTypeId	integer NOT NULL,
+Price	integer NOT NULL,
+Title		varchar(80) not null,
+ProdDesc		varchar(80) not null,
+Quantity		integer not null,
+Constraint FK_ProductType FOREIGN KEY(ProductTypeId) REFERENCES ProductType(Id)
```
Comment (Rian501) [inline BangazonDB.sql]: A product needs a foreign key of the customer who is selling it.

File: BangazonDB.sql [hunk 2]
```diff
@@ -0,0 +1,552 @@
...
+insert into PaymentType
+(AccountNo, AccType, Nickname, CustomerId) 
+select 60000, 'CC', 'April MasterCard', c.Id
+from Customer c where c.FirstName = 'April' and c.LastName = 'Watson';
+
+Create table Orders (
+Id		integer not null primary key IDENTITY,
+OrderDate		varchar(80) not null,
+CustomerId 		integer not null,
+PaymentTypeId	integer not null,
```
Comment (Rian501) [inline BangazonDB.sql]: Payment type must not be not null.

File: BangazonDB.sql (hunk 1 above)
Comment (aprilrochelle) [reply BangazonDB.sql]: It's giving me an error that the Customer table is invalid now.

**Pull request closed without merging**